lib/react_client.dart

```diff
@@ -675,6 +680,7 @@ class ReactDomComponentFactoryProxy extends ReactComponentFactoryProxy {
   ReactDomComponentFactoryProxy(name)
       : this.name = name,
         this.factory = React.createFactory(name) {
+    // TODO: Should we remove this once we validate that the bug is gone in Dart 2 DDC?
```

Comment:
FYI This has been validated already and does need to be removed. We can do this in a separate PR